config/gni/devtools_grd_files.gni

```diff
@@ -190,6 +190,7 @@ grd_files_release_sources = [
   "front_end/Images/review.svg",
   "front_end/Images/right-panel-close.svg",
   "front_end/Images/right-panel-open.svg",
+  "front_end/Images/rnIcon.png",
```

Comment:
Let's maybe put all rn-specific images in a subdirectory for ease of maintenance?